eng/readytorun.targets

```diff
@@ -0,0 +1,33 @@
+<Project>
+  <PropertyGroup>
+    <!-- Only use mibc files if UsingToolIbcOptimization is false. Allows enabling/disabling using ibc instead of mibc data -->
+    <IncludePgoFiles Condition="'$(UsingToolIbcOptimization)' != 'true' and '$(EnableNgenOptimization)' == 'true' and '$(DotNetBuildFromSource)' != 'true'">true</IncludePgoFiles>
+  </PropertyGroup>
+
+  <Target Name="ResolveReadyToRunCompilers" DependsOnTargets="ResolveRuntimeFilesFromLocalBuild">
+    <ItemGroup>
+      <OptimizationMibcFiles Include="$(CoreCLRArtifactsPath)StandardOptimizationData.mibc" Condition="'$(DotNetBuildFromSource)' != 'true'" />
+    </ItemGroup>
```

Comment:
AFAIK this item is only used by crossgen-corelib.proj. Wouldn't it make more sense to move it there?

Reply:
It’s used when publishing the shared framework as a whole as well.